impress-org/give-simple-social-shout
# Fix: Missing social icons

Resolves #16 

## Description

This PR resolves the issue with missing icons. The issue was that icons were loaded by providing image URL to the PHP `file_get_contents` function. On servers that have disabled [allow_url_fopen](https://www.php.net/manual/en/filesystem.configuration.php#ini.allow-url-fopen), `file_get_contents` cannot load remote files. 

The issue is resolved by using `require` instead of `file_get_contents`.

## Testing Instructions

1. Create a Donation form that uses the Legacy template
2. Setup and enable Simple Social Shout 
3. Make a donation

On Donation Confirmation screen you should see all social icons displayed correctly 



## Pre-review Checklist
